webapp/src/containers/SwapPage/service.tsx

```diff
@@ -286,27 +286,27 @@ export const handlePoolSwap = async (formState) => {
 
   store.dispatch(poolSwapRefreshUTXOSuccess());
 
-  if (address1 !== address2) {
-    const txId1 = await rpcClient.sendToAddress(
-      address2,
-      DEFAULT_DFI_FOR_ACCOUNT_TO_ACCOUNT,
-      true
-    );
-    const txId2 = await rpcClient.sendToAddress(
-      address1,
-      DEFAULT_DFI_FOR_ACCOUNT_TO_ACCOUNT,
-      true
-    );
-    await getTransactionInfo(txId1);
-    await getTransactionInfo(txId2);
-  } else {
-    const txId = await rpcClient.sendToAddress(
-      address1,
-      DEFAULT_DFI_FOR_ACCOUNT_TO_ACCOUNT,
-      true
-    );
-    await getTransactionInfo(txId);
-  }
+  // if (address1 !== address2) {
+  //   const txId1 = await rpcClient.sendToAddress(
+  //     address2,
+  //     DEFAULT_DFI_FOR_ACCOUNT_TO_ACCOUNT,
+  //     true
+  //   );
+  //   const txId2 = await rpcClient.sendToAddress(
+  //     address1,
+  //     DEFAULT_DFI_FOR_ACCOUNT_TO_ACCOUNT,
+  //     true
+  //   );
+  //   await getTransactionInfo(txId1);
+  //   await getTransactionInfo(txId2);
+  // } else {
+  //   const txId = await rpcClient.sendToAddress(
+  //     address1,
+  //     DEFAULT_DFI_FOR_ACCOUNT_TO_ACCOUNT,
+  //     true
+  //   );
+  //   await getTransactionInfo(txId);
+  // }
```

Comment:
is this section commented out to remove refresh utxo? can we delete instead of comment please

Reply:
Will do